transformer_engine/common/include/transformer_engine/fused_attn.h

```diff
@@ -248,6 +256,9 @@ void nvte_fused_attn_fwd_qkvpacked(const NVTETensor QKV, const NVTETensor Bias,
  *  \param[in]     qkv_layout               QKV tensor's layout.
  *  \param[in]     bias_type                Bias type.
  *  \param[in]     attn_mask_type           Attention mask type.
+ *  \param[in]     window_size_left         Sliding window size (the left half).
+ *  \param[in]     window_size_right        Sliding window size (the right half).
+ *  \param[in]     deterministic             Whether to execute with deterministic behaviours.
```

Comment:
typo: one more space
```suggestion
 *  \param[in]     deterministic            Whether to execute with deterministic behaviours.
```

Reply:
Done